va/validation-authority.go

```diff
@@ -133,25 +143,55 @@ func (va ValidationAuthorityImpl) validateSimpleHTTP(identifier core.AcmeIdentif
 	}
 
 	httpRequest.Host = hostName
-	tr := &http.Transport{
-		// We are talking to a client that does not yet have a certificate,
-		// so we accept a temporary, invalid one.
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
-		// We don't expect to make multiple requests to a client, so close
-		// connection immediately.
-		DisableKeepAlives: true,
-	}
+	tr := *http.DefaultTransport.(*http.Transport)
```

Comment:
This whole section is too verbose.  Why not follow the good example in the `net/http` documentation and do something like:

```
tr := &http.Transport{
  TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
  // etc.
}
```